core-bundle/contao/dca/tl_content.php

```diff
@@ -50,8 +50,11 @@
 			'keys' => array
 			(
 				'id' => 'primary',
+				'tstamp' => 'index',
+				'sorting' => 'index',
 				'pid,ptable,invisible,start,stop' => 'index',
-				'type' => 'index'
+				'type' => 'index',
+				'ptable' => 'index'
```

Comment:
```suggestion
				'type' => 'index'
```
We don’t need an extra index for `ptable`, if we fix the order of the multi-column-index above.